src/Init.php

```diff
@@ -192,6 +194,8 @@ function feature($name)
 // Usecases that perform searches are the most typical usage of data objects.
 $di->params['Ushahidi\Factory\DataFactory']['actions'] = [
 	'search' => $di->lazyNew('Ushahidi\Core\SearchData'),
+	'update' => $di->lazyNew('Ushahidi\Core\SearchData'),
```

Comment:
If we make `bulk_update` and `bulk_delete` distinct actions this would end up making more sense..